qaoa/initialstates/lessthank_initialstate.py

```diff
@@ -6,13 +6,44 @@
 
 
 class LessThanK(InitialState):
+    """
+    LessThanK initial state.
+
+    Subclass for the `InitialState` class, and it creates a quantum circuit that creates the initial state for the for the MAX k-CUT problem for
+    different cases of k subsets.
+
+    Attributes:
+        k (int): subsets (or "colors") of the vertices in the MAX k-CUT problem is seperated into
+
+    Methods:
+        create_circuit(): creates a circuit that can create the wanted initial state for the special k
+        is_power_of_two_or_between_2_and_8(): checks that the k is either to the power of 2 or between 2 and 8
+        power_of_two(): creates a circuit for k is to the power of 2
+        k3(): creates a circuit for k = 3
```

Comment:
These are private functions and are never called from outside the class (I think). Could you make the change?